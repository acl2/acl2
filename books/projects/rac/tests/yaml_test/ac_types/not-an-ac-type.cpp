// RAC begin

int foo(int a) {
  return a.slc<4>(0);
}
