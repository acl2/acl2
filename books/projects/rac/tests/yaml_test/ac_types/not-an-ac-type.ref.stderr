not-an-ac-type.cpp:4 (9-10): Base (of type int) is not a register
 4 |   return a.slc<4>(0);
   |          ^
